src/controllers/hid/hidcontroller.cpp

```diff
@@ -295,6 +239,29 @@ int HidController::close() {
     return 0;
 }
 
+bool HidController::poll() {
+    Trace hidRead("HidReader poll");
+
+    for (int i = 0; i < 10; i++) {
```

Comment:
Use a constant instead of some magic number somewhere in the code.

Reply:
What is the significance of using 10 here?

Reply:
No significance, just a random number to break out at in case we reach > 10 packets in 1 poll() call.